### PR TITLE
event and attendee table

### PR DESCRIPTION
**Pull request by afyacy**
Schema updated

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -11,15 +11,31 @@ generator client {
...
+  company   Company? @relation(fields: [companyId], references: [companyId])
   email         String?   @unique
   name          String
   password      String
   @@map(name: "users")
+}
+
+model Event {
+  eventId        Int       @default(autoincrement()) @id
+  date     DateTime
```
Comment (simbamentor) [inline prisma/schema.prisma:31]: can you add a `startDate` and an `endDate` instead?

File: prisma/schema.prisma [hunk 2]
```diff
@@ -11,15 +11,31 @@ generator client {
...
+  name         String
+  user          String
+  attendee      String
```
Comment (simbamentor) [inline prisma/schema.prisma:34]: `attendee` should be a reference to the attendee table

File: prisma/schema.prisma [hunk 3]
```diff
@@ -11,15 +11,31 @@ generator client {
...
+}
+
+model Attendee {
+  attendeeId        Int       @default(autoincrement()) @id
+  email         String 
+  name          String
```
Comment (simbamentor) [inline prisma/schema.prisma:40]: every attendee belongs to one event

File: prisma/schema.prisma (hunk 2 above)
Comment (simbamentor) [reply prisma/schema.prisma:34]: an event can have multiple attendees

File: prisma/schema.prisma [hunk 4]
```diff
@@ -11,15 +11,31 @@ generator client {
 }
 
 model Company {
-  companyId        Int     @default(autoincrement()) @id
+  companyId        Int   @default(autoincrement()) @id
   name     String
+  users User[]
```
Comment (simbamentor) [inline prisma/schema.prisma:16]: can you improve the formatting in this file?

Int
Int
String
User[]

should be on the same vertical line

**Review by simbamentor — CHANGES_REQUESTED** [review]
added comments

File: prisma/schema.prisma (hunk 1 above)
Comment (afyacy) [reply prisma/schema.prisma:31]: done

File: prisma/schema.prisma (hunk 2 above)
Comment (afyacy) [reply prisma/schema.prisma:34]: done

File: prisma/schema.prisma (hunk 4 above)
Comment (afyacy) [reply prisma/schema.prisma:16]: Done

**Pull request merged**